src/version-2-0-0.md

```diff
@@ -0,0 +1,26 @@
+Date: 2016-03-08
+Title: Release 2.0.0
+Tags: news, release notes
+Slug: uniqush-push-2-0-0
+Category: news
+
+Please check out our [release notes] for version 2.0.0.
+
+We've improved the response format of most APIs. Instead of sending server logs
+back, they now return JSON response objects. This makes it easy to distinguish
+successes from failures, since, for example, a subscriber can have many
+delivery points and not all of them may succeed.
+
+The APNS module has been updated to support the new 2048 byte (2kB) limit.
+
+There are a few bugfixes for a couple of memory leaks and with closing
```

Comment:
Awkward phrasing. "There are a few bugfixes, fixing both memory leaks and improperly closed connections" (Or split up into two sentences)